src/navigation/screens/Profile.tsx

```
@@ -1,21 +1,144 @@
 import React from 'react';
-import { Text } from '@react-navigation/elements';
-import { StyleSheet, View } from 'react-native';
-
+import { StyleSheet, View, Image, Text, TouchableOpacity, Switch, SafeAreaView, ScrollView } from 'react-native';
```

ESLint

[error] 'StyleSheet' is defined but never used.

(@typescript-eslint/no-unused-vars)